pkg/vex/vex.go

```diff
@@ -70,15 +70,19 @@ type Metadata struct {
 	Author string `json:"author"`
 
 	// AuthorRole describes the role of the document Author.
-	AuthorRole string `json:"role"`
+	AuthorRole string `json:"role,omitempty"`
 
 	// Timestamp defines the time at which the document was issued.
 	Timestamp *time.Time `json:"timestamp"`
 
+	// LastUpdated marks the time when the document had its last update. When the
+	// document changes both version and this field should be updated.
+	LastUpdated *time.Time `json:"last_updated,omitempty"`
+
 	// Version is the document version. It must be incremented when any content
 	// within the VEX document changes, including any VEX statements included within
 	// the VEX document.
-	Version string `json:"version"`
+	Version int `json:"version"`
```

Comment:
since we're making some breaking changes, should we introduce the VEX spec version as well? https://github.com/openvex/spec/issues/20

Reply:
I had added the version but I think I lost the commit at some point. It's back! Documents now have the spec context versioned:
```
go run . create "pkg:apk/wolfi/git@2.39.0-r1?arch=x86_64" CVE-2022-39260 fixed 
```
```json

{
  "@context": "https://openvex.dev/ns/0.1.0",
  "@id": "https://openvex.dev/docs/public/vex-d1d09738f646225c75aa64ef57b5c669947b6d5b1ad2ffa03816eba43e96e372",
  "author": "Unknown Author",
  "timestamp": "2023-07-13T18:36:48.978656084-06:00",
  "version": 1,
  "statements": [
    {
      "vulnerability": {
        "name": "CVE-2022-39260"

```